pkg/nsx/services/ipblocksinfo/store.go

```diff
@@ -96,3 +98,37 @@ func (vs *VPCStore) Apply(i interface{}) error {
 	}
 	return nil
 }
+
+type VpcAttachmentStore struct {
```

Comment:
Would you think we can delete VpcStore in this file as it is not used? Maybe also change the corresponding unit test in store_test.go for VpcStore and VpcAttachmentStore

Reply:
yes

Reply:
Done

Reply:
Are you going to delete VpcStore in this file? It's still here now.

Reply:
I was dizzy before. the VpcAttachmentStore is used in ipblocksinfo but not used in vpc. I removed that in vpc/store.go